raiden-dapp/src/components/dialogs/UdcDepositDialog.vue

```diff
@@ -135,7 +136,27 @@ export default class UdcDepositDialog extends Vue {
   }
 
   get valid(): boolean {
-    return /^[1-9]\d*$/.test(this.defaultUtilityTokenAmount);
+    let utilityTokenAmount: BigNumber;
+
+    try {
+      utilityTokenAmount = parseUnits(
+        this.defaultUtilityTokenAmount,
+        this.udcToken.decimals
+      );
+    } catch (err) {
+      utilityTokenAmount = Zero;
```

Comment:
Maybe if it fails to parse, you should directly `return false;`?

Reply:
Yes, I'll do so.